internal/buffer/search.go

```diff
@@ -65,17 +63,17 @@ func (b *Buffer) findDown(r *regexp.Regexp, start, end Loc) ([2]Loc, bool) {
 	rPadded := padRegexp(r)
 
 	for i := start.Y; i <= end.Y; i++ {
-		l, charpos, ri := findLineParams(b, start, end, i)
+		l, charpos, padMode := findLineParams(b, start, end, i)
 
-		match := rPadded[ri].FindIndex(l)
+		match := rPadded[padMode].FindIndex(l)
 
 		if match != nil {
 			start := Loc{charpos + util.RunePos(l, match[0]), i}
-			if ri == rPadStart || ri == rPadBoth {
+			if padMode & padStart != 0 {
```

Comment:
gofmt doesn't want spaces here (not that I like it)...

Reply:
added